plugin: add Cloudsmith migration changes (#77)
ConsenSys/doc.goquorum: docs/Concepts/Plugins/Plugins.md

    Using `--plugins.skipverify` introduces security risks and isn't recommended for production environments.

!!! important

>     Before `21.4.1`, Quorum's default configuration is to use Bintray as the Plugin Central Server to distribute the official plugins.

Reviewer: ```suggestion
    Before GoQuorum `21.4.1`, the default Plugin Central Server configuration used Bintray to distribute the official plugins.
```
Simplify language
JPM Quorum is now GoQuorum

>     As Bintray is into sunset on May 1st 2020, please [configure](../../HowTo/Configure/Plugins.md#plugincentralconfiguration) plugin central as below 

Reviewer: ```suggestion
    As Bintray will stop working on May 1st 2020, [configure plugin central](../../HowTo/Configure/Plugins.md#plugincentralconfiguration) to use ConsenSys Cloudsmith repository
```
Simplify language and prevent the use of positional words.
Be clear about the new repos being on Cloudsmith
